test/unittests/commands/excel2xml/test_excel2xml_lib.py

```diff
@@ -406,11 +407,14 @@ def test_make_integer_prop(self) -> None:
         self.run_test(prop, method, [int(x) for x in different_values], invalid_values)
 
     def test_make_interval_prop(self) -> None:
```

Comment:
`<interval-prop>`s can now only have one value, while all other props can have multiple values. Thus, the usual testing pattern cannot be applied any more.